reliability-engineering/pipelines/performance-platform.yml

```diff
@@ -0,0 +1,84 @@
+resources:
+  - name: stagecraft-git
+    type: git
+    source:
+      uri: https://github.com/alphagov/stagecraft.git
+      branch: change_deploy_for_concourse
+
+jobs:
+  - name: deploy-stagecraft-to-paas-staging
+    serial: true
+    plan:
+      - get: stagecraft-git
+        trigger: true
+      - task: deploy-stagecraft-to-paas-staging
+        timeout: 15m
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: governmentpaas/cf-cli
+          inputs:
+            - name: stagecraft-git
+              path: repo
+          params:
+            APP_SECRET_KEY: ((app_secret_key_staging))
+            APP_FERNET_KEY: ((app_fernet_key))
+            FLOWER_BASIC_AUTH: ((flower_basic_auth_staging))
+            PAAS_USER: ((cf_user))
+            PAAS_PASSWORD: ((cf_password))
+            SIGNON_CLIENT_ID: ((signon_client_id))
+            GOVUK_APP_DOMAIN: 'staging.publishing.service.gov.uk'
+            GOVUK_WEBSITE_ROOT: 'https://www-origin.staging.publishing.service.gov.uk'
+            BACKDROP_PUBLIC_DOMAIN: 'performance-platform-backdrop-read.cloudapps.digital'
+            REDIS_DATABASE_NUMBER: '0'
+            STAGECRAFT_COLLECTION_ENDPOINT_TOKEN: ((stagecraft_collection_endpoint_token))
+            DJANGO_SETTINGS_MODULE: 'stagecraft.settings.staging'
+          run:
+            path: sh
+            dir: repo
+            args:
+              - -c
+              - |
+                chmod +x etc/deploy.sh
```

Comment:
We could `chmod` this file and commit it into the Stagecraft repo to avoid having to do this here.

Reply:
This has been done and merged, so I have removed the chmod..